decidim-core/lib/decidim/content_parsers/user_parser.rb

```diff
@@ -35,19 +35,17 @@ def metadata
         Metadata.new(existing_users)
       end
 
-      private
```

Comment:
Any reason why making these methods public?

Reply:
absolutely no one, i'm fixing this in my next commit